images/linux/scripts/helpers/install.sh

```diff
@@ -23,12 +23,16 @@ download_with_retries() {
     i=20
     while [ $i -gt 0 ]; do
         ((i--))
-        eval $COMMAND
+        echo "Verifying $URL HTTP status code..."
+        verify_http_status_code $URL
         if [ $? != 0 ]; then
```

Comment:
why the condition is for non-zero exit code? Do we want to download the file from broken URLs only? 😀